internal/handlers/tigris/msg_getlog.go

```diff
@@ -83,7 +84,9 @@ func (h *Handler) MsgGetLog(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		))
 
 	case "startupWarnings":
-		info, err := h.db.Driver.Info(ctx)
```

Comment:
internal/handlers/tigris/msg_getlog.go:86:9: shadow: declaration of "err" shadows declaration at line 36 (govet)
                info, err := h.db.Driver.Info(ctx)